### PR TITLE
feat(graphql): use session

### PR DESCRIPTION
**Pull request by moreal**
This pull request makes the launcher client to call `mutation.auth.login` while sharing session between `main` and `renderer` process.

### CODE REVIEW

**Comment by moreal** [comment]
Could you review this pull request, reviewers?

File: src/main/headless.ts [hunk 1]
```diff
@@ -120,7 +126,19 @@ class Headless {
     const body = JSON.stringify({
       PrivateKeyString: privateKey,
     });
-    return this.retriableFetch("set-private-key", body);
+    const httpResult = await this.retriableFetch("set-private-key", body);
```
Comment (longfin) [inline src/main/headless.ts:129]: Just curious, do we still need `set-private-key` endpoint?

**Review by longfin — APPROVED** [review]

File: src/main/headless.ts (hunk 1 above)
Comment (moreal) [reply src/main/headless.ts:129]: At this time, yes. It is for mining. To remove it, https://github.com/planetarium/NineChronicles.Headless/pull/413 pull request should be merged.

Comment (longfin) [reply src/main/headless.ts:129]: 👌

**Review by area363 — APPROVED** [review]

**Pull request merged**